### PR TITLE
[type:feature] Optimize SQL files for resource,permission

### PR DESCRIPTION
**Pull request by nuo-promise**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/spring/ResourceAndPermissionDataSourceLoader.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        }
+    }
+
+    /**
+     * get resource info by plugin title.
+     * @param pluginTitle plugin title
+     * @return {@link Boolean}
+     */
+    private Boolean getResource(final String pluginTitle) {
+        return ObjectUtils.notEqual(resourceService.findByTitle(pluginTitle), null);
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/spring/ResourceAndPermissionDataSourceLoader.java:113]: use Objects

File: shenyu-register-center/shenyu-register-client/shenyu-register-client-api/pom.xml [hunk 2]
```diff
@@ -36,6 +36,10 @@
             <artifactId>shenyu-spi</artifactId>
             <version>${project.version}</version>
         </dependency>
-    </dependencies>
+		<dependency>
```
Comment (yu199195) [inline shenyu-register-center/shenyu-register-client/shenyu-register-client-api/pom.xml:39]: why add this？

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/spring/ResourceAndPermissionDataSourceLoader.java (hunk 1 above)
Comment (nuo-promise) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/spring/ResourceAndPermissionDataSourceLoader.java:113]: OK

**Review by yu199195 — APPROVED** [review]

File: shenyu-register-center/shenyu-register-client/shenyu-register-client-api/pom.xml (hunk 2 above)
Comment (nuo-promise) [reply shenyu-register-center/shenyu-register-client/shenyu-register-client-api/pom.xml:39]: Prompt motan for a class not found

**Review by yu199195 — APPROVED** [review]

**Pull request merged**